tensorflow/c/eager/c_api.h

```diff
@@ -81,6 +81,17 @@ TF_CAPI_EXPORT extern void TFE_ContextOptionsSetAsync(TFE_ContextOptions*,
 TF_CAPI_EXPORT extern void TFE_ContextOptionsSetDevicePlacementPolicy(
     TFE_ContextOptions*, TFE_ContextDevicePlacementPolicy);
 
+// Infer the types of the single input arguments into their respective type
+// attribute (default is false).
+//
+// This replicates the behaviour of the NodeDefBuilder in graph mode. It is
+// assumed that inputs are added to the operaton in the same order as they are
+// declared in its definition.
+// It is also assumed that inferred attributes cannot be set explicitly when
```

Comment:
I did not see why this is a problem. Your code seems to just override if something is already set.

Reply:
See previous comments, those "warnings" will be removed.